packages/features/bookings/Booker/Booker.tsx

```diff
@@ -72,9 +77,28 @@ const BookerComponent = ({
     shallow
   );
 
+  const date = dayjs(selectedDate).format("YYYY-MM-DD");
+  const schedule = useScheduleForEvent({ prefetchNextMonth: true });
+  const nonEmptyScheduleDays = useNonEmptyScheduleDays(schedule?.data?.slots).filter(
+    (slot) => dayjs(selectedDate).diff(slot, "day") <= 0
+  );
+
   const extraDays = isTablet ? extraDaysConfig[layout].tablet : extraDaysConfig[layout].desktop;
   const bookerLayouts = event.data?.profile?.bookerLayouts || defaultBookerLayoutSettings;
   const animationScope = useBookerResizeAnimation(layout, bookerState);
+  const addonDays =
+    nonEmptyScheduleDays.length < extraDays ? (extraDays - nonEmptyScheduleDays.length + 1) * 7 : 0;
+  const availableSlots = nonEmptyScheduleDays.slice(0, extraDays + 1);
```

Comment:
```suggestion
  const extraDaysOffset = extraDays + 1; // Reason why this offset it's needed or what it does.
  const availableSlots = nonEmptyScheduleDays.slice(0, extraDaysOffset);
```
Seeing a lot of + 1's we are better of declaring a name for that and a comment will help others in the future.

Reply:
extraDays + 1 is used only once in the whole code. I though if it's used more than one, then it's obvious to declare a variable. I feel it's better to go with comment line describing the purpose.